j5/boards/sr/__init__.py

```diff
@@ -0,0 +1 @@
+"""Boards made by Student Robotics."""
```

Comment:
At one point there were discussions of having competition specific implementations in their own packages. This would mean a smaller 'core' j5 and consumers can then specify which specific implementations they wish to include in their projects.

This is not a huge deal right now, but we should decide whether this is something we still want to do and maybe open an issue to track it.

Reply:
@kierdavis and I discussed this a couple of weeks ago and concluded that we should keep things together for now. In the future, I definitely think that things should be separate though and would actually be happier if they were separated now. Unix philosophy and all that. Besides, the whole point of `j5` is that it is modular.

I have opened #97 to discuss this further.